lemur/notifications/service.py

```diff
@@ -104,12 +104,13 @@ def create(label, plugin_name, options, description, certificates):
     return database.create(notification)
 
 
-def update(notification_id, label, options, description, active, certificates):
+def update(notification_id, label, plugin_name, options, description, active, certificates):
```

Comment:
Function: Changing the plugin selected required updating the `plugin_name` field as well as the options.